tests/js-bigint/in/js-bigint/index.d.ts

```diff
@@ -0,0 +1,3 @@
+interface A {
```

Comment:
Actually you could just add an interface to the bottom of the [bigint test file](https://github.com/ScalablyTyped/Converter/blob/master/tests/bigint/in/bigint/index.d.ts).
something like:
```typescript
interface Test {
    a: bigint,
    b: BigInt,
    c: BigInt.BigInt
}

You'll still need to add `BigInt` to the corresponding std file.